src/dom/index.js

```diff
@@ -85,7 +85,10 @@ export function setAccessor(node, name, old, value, isSvg) {
 	}
 	else {
 		let ns = isSvg && (name !== (name = name.replace(/^xlink:?/, '')));
-		if (value==null || value===false) {
+		// spellcheck is treated differently than all other boolean values and
+		// should not be removed when the value is `false`. See:
+		// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-spellcheck
+		if ((value==null || value===false) && name !== "spellCheck") {
```

Comment:
I think the name is `"spellcheck"`, right?  In that case it actually wouldn't fall through to here, but rather would pass the `in` check on line 82.  It seems like the special-casing belongs there:

```js
// replaces L84:
if ((value==null || value===false) && name!='spellcheck') node.removeAttribute(name);
```

Reply:
Went the `setAttribute` way, but forcing lowercase seems better. Will change that.